README.md

```diff
@@ -0,0 +1,22 @@
+# mygnuboy
+My own version of gnuboy emulator
```

Comment:
i dont think we want to change the name to "mygnuboy". it should stay as close to laguna's original as possible.